src/Libraries/Microsoft.Extensions.Telemetry/Logging/ExtendedLogger.cs

```diff
@@ -188,13 +188,11 @@ private void ModernPath(LogLevel logLevel, EventId eventId, LoggerMessageState m
         var loggers = MessageLoggers;
         var config = _factory.Config;
 
-        List<Exception>? exceptions = null;
-
         // redact
         JustInTimeRedactor? jitRedactors = null;
-        for (int i = 0; i < msgState.NumClassifiedTags; i++)
+        if (msgState.NumClassifiedTags > 0)
```

Comment:
Is it faster to have an `if` statement before the `for` loop? If not, then we don't really need this `if` - it will be covered by `i < msgState.NumClassifiedTags` in the `for` loop

Reply:
Whoops, that was a leftover from when I had something explicitly in the if block, but not in the loop. I removed the if.

Thanks.